src/global/common.rs

```diff
@@ -478,6 +480,29 @@ pub(crate) async fn get_expose_scripts_sync_status(
     Ok((to_remove, to_add))
 }
 
+/// Check if they were all auto-exposed, or if the user manually exposed a subset of them
+pub fn check_auto_exposed(
```

Comment:
Technically, the user could have manually exposed all of them :)
Maybe call the function `check_all_exposed`? And adapt the docstring accordingly

Reply:
good wording catch